assets/js/elements-utils.js

```diff
@@ -10,3 +10,12 @@ export function removeChildren(node){
         node.removeChild( node.firstChild );
     }
 }
+
+export function makeCopyAllBadgesButtonElement(badgesAsMarkdown) {
+    let div = document.createElement("div");
```

Comment:
you couldn't use the `makeElement()` utils?

Reply:
:+1: